agentai/vectordb.py

```diff
@@ -0,0 +1,39 @@
+import chromadb
+from parsers import Parser
+import uuid
+from pydantic import BaseModel, Field
+from typing import List, Optional
+from agentai.annotations import tool, ToolRegistry
+
+
+class Query(BaseModel):
+    query_embedding: List[int] = Field(..., description="Embedding for the query to srach")
+    query_text: Optional[str] = Field(..., description="Simplified query from the user to search")
+    k: str = Field(..., description="The number of results requested")
```

Comment:
```suggestion
class Query(BaseModel):
    query_embedding: Optional[List[int]] = Field(..., description="Embedding for the query to search")
    query_text: Optional[str] = Field(..., description="Simplified query from the user to search")
    k: int = Field(..., description="The number of results requested")
```

Query Embedding can possible be created by the library and not be part of the user query. Why is a number `str`?

Reply:
Sorry sorry. Took the pseudo code I wrote and forgot to change this.